### ISSUE TITLE
Show allowed values on bad filter parameter

### ISSUE THREAD

**Issue by jrmi** [tortoise/tortoise-orm#63]
When I use an inexistent parameter in a query filter, I get this error message: 
```python
tortoise.exceptions.FieldError: unknown filter param non_existing_attribute
```
Is it possible to show allowed values to ease debugging like:
> unknown filter parameter 'non_existing_attribute'. Allowed values are ['attribute1', 'attribute2', ...].

Another possibility is to clarify error message with something like:

> You can only filter on existing attributes for the <Tournament> model. <non_existing_parameter> is invalid.

**Comment by grigi** [comment]
A comprehensive error is like so:
```py
tortoise.exceptions.FieldError: Unknown filter param 'charup'. Allowed values are ['char', 'char__contains', 'char__endswith', 'char__gt', 'char__gte', 'char__icontains', 'char__iendswith', 'char__in', 'char__isnull', 'char__istartswith', 'char__lt', 'char__lte', 'char__not', 'char__not_in', 'char__not_isnull', 'char__startswith', 'char_null', 'char_null__contains', 'char_null__endswith', 'char_null__gt', 'char_null__gte', 'char_null__icontains', 'char_null__iendswith', 'char_null__in', 'char_null__isnull', 'char_null__istartswith', 'char_null__lt', 'char_null__lte', 'char_null__not', 'char_null__not_in', 'char_null__not_isnull', 'char_null__startswith', 'id', 'id__contains', 'id__endswith', 'id__gt', 'id__gte', 'id__icontains', 'id__iendswith', 'id__in', 'id__isnull', 'id__istartswith', 'id__lt', 'id__lte', 'id__not', 'id__not_in', 'id__not_isnull', 'id__startswith']
```
Its a very verbose, and that is for 3 fields only... But correct...

A version showing the base fields:
```py
tortoise.exceptions.FieldError: Unknown filter param 'charup'. Allowed base values are ['char', 'char_null', 'id']
```
And much shorter, but a bit of a lie, as we don't show the filters...
To be even more helpful we could show the fields filters if you use an invalid filter?
@Zeliboba5 What do you think?

I also realised that the set of eligible values contains filters that make no sense, e.g. `__is_null` on non-nullable fields, `__contains` `__beginswith` etc on integers? So we could reduce that set by a bit.

**Comment by jrmi** [comment]
I was thinking more about the second option showing only base fields. "subfilters" can be added and deduced from this list by the user. I think the important thing is that the user can easily correct is code after seeing this message.

**Comment by grigi** [comment]
Thanks @jrmi For reporting on these "paper cut" level issues. :+1: It is a thing that OSS projects often suffer on.
This will be fixed for v0.10.10

**Comment by FIRDOUS-BHAT** [comment]
> A comprehensive error is like so:
> 
> ```python
> tortoise.exceptions.FieldError: Unknown filter param 'charup'. Allowed values are ['char', 'char__contains', 'char__endswith', 'char__gt', 'char__gte', 'char__icontains', 'char__iendswith', 'char__in', 'char__isnull', 'char__istartswith', 'char__lt', 'char__lte', 'char__not', 'char__not_in', 'char__not_isnull', 'char__startswith', 'char_null', 'char_null__contains', 'char_null__endswith', 'char_null__gt', 'char_null__gte', 'char_null__icontains', 'char_null__iendswith', 'char_null__in', 'char_null__isnull', 'char_null__istartswith', 'char_null__lt', 'char_null__lte', 'char_null__not', 'char_null__not_in', 'char_null__not_isnull', 'char_null__startswith', 'id', 'id__contains', 'id__endswith', 'id__gt', 'id__gte', 'id__icontains', 'id__iendswith', 'id__in', 'id__isnull', 'id__istartswith', 'id__lt', 'id__lte', 'id__not', 'id__not_in', 'id__not_isnull', 'id__startswith']
> ```
> 
> Its a very verbose, and that is for 3 fields only... But correct...
> 
> A version showing the base fields:
> 
> ```python
> tortoise.exceptions.FieldError: Unknown filter param 'charup'. Allowed base values are ['char', 'char_null', 'id']
> ```
> 
> And much shorter, but a bit of a lie, as we don't show the filters...
> To be even more helpful we could show the fields filters if you use an invalid filter?
> @Zeliboba5 What do you think?
> 
> I also realised that the set of eligible values contains filters that make no sense, e.g. `__is_null` on non-nullable fields, `__contains` `__beginswith` etc on integers? So we could reduce that set by a bit.

@grigi 
I'm somewhat the same error,
  `tortoise.exceptions.FieldError: Unknown filter param 'in'. Allowed base values are ...`
  
  while running this query,
  
  `lectures = await CourseCategoryLectures.filter(category_topic__in=uid)`
